app.test.js

```diff
@@ -127,4 +127,74 @@ describe("GET /api/articles", () => {
             expect(resBody).toBeSortedBy("created_at", {descending : true})
         })
     })
-})
\ No newline at end of file
+})
+
+describe("GET /api/articles/:article_id/comments", () => {
+    test("returns an error when given article id is not a number type", () => {
+        return request(app)
+        .get("/api/articles/notANumber/comments")
+        .expect(400)
+        .then(({body}) => {
+            expect(body.message).toBe("invalid id path")
+        })
+    })
+    
+    test("returns an error when given an article id path that does not exist", () => {
+        return request(app)
+        .get("/api/articles/999/comments")
+        .expect(404)
+        .then(({body}) => {
+            expect(body.message).toBe("article not found")    
+        })
+    })
+
+    test("returns an empty array when the given article id does not have any comments.", () => {
+        return request(app)
+        .get("/api/articles/2/comments")
+        .expect(200)
+        .then(({body}) => {
+            expect(body.comments).toHaveLength(0)
+        })
+    })
+
+    test("returns a single object array when the article only has one comment.", () => {
```

Comment:
This test might be in the realm of over-testing, good to have but not really necessary